### DISCUSSION TITLE
容器终端超时导致登录状态退出问题

### DISCUSSION THREAD

**Discussion by JAINKRE** [1Panel-dev/1Panel#5977] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

首先容器的终端连接时间过短

其次，通过连接容器的终端，如果超时，会导致面板的登录状态退出。

但这个问题好像是概率问题，至少已经遇到了好几次。

### 重现步骤

容器终端因为下载和校验模型过长，导致超时：
![PixPin_2024-07-29_15-16-08](https://github.com/user-attachments/assets/2fc41fc9-2507-4f48-85dd-6e787279895a)

在刷新页面，面板直接退出需要重新登录：
![PixPin_2024-07-29_15-16-20](https://github.com/user-attachments/assets/43d7aae1-efa0-4bc5-9dd8-69d9bf9a514b)

**Comment by ssongliu** [comment]
感谢反馈，麻烦提供一下面板日志信息

**Comment by JAINKRE** [comment]
> 感谢反馈，麻烦提供一下面板日志信息

从哪里看这个日志

**Comment by ssongliu** [comment]
`日志审计` - `系统日志`

**Comment by JAINKRE** [comment]
> `日志审计` - `系统日志`

ok 可否提供一下邮箱，私发给你

**Comment by ssongliu** [comment]
liu.song@fit2cloud.com
你有加社区群吗？群里吱一声，我直接加你。
